src/lib/openapi/spec/resource-limits-schema.ts

```diff
@@ -103,6 +104,13 @@ export const resourceLimitsSchema = {
             example: 300,
             description: 'The maximum number of segments allowed.',
         },
+        flags: {
```

Comment:
Should we use `flags` or `featureFlags`; got any thoughts? I put `flags` in because it's shorter, but maybe `featureFlags` is more explicit? 🤷🏼